coreels/settings.py

```diff
@@ -66,7 +67,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': ['templates'],
+        'DIRS': [os.path.join(BASE_DIR,'templates')],
```

Comment:
We are using Django 4.* which qualifies for the new PATH functions that comes with python 3.

Hence, you can use `BASE_DIR / templates'` rather than `os.path.join(BASE_DIR, 'templates')`

[Checkout Django Documentation](https://docs.djangoproject.com/en/4.0/) for more details.